include/bzfsAPI.h

```diff
@@ -1460,6 +1461,20 @@ class BZF_API bz_PermissionModificationData_V1 : public bz_EventData
     bool customPerm;
 };
 
+class BZF_API bz_ReloadData_V1 : public bz_EventData
+{
+public:
+    bz_ReloadData_V1() : bz_EventData(bz_eReloadEvent)
+        , playerID(-1)
+        , target("all")
+        , handled(false)
+    {}
```

Comment:
why is this in-line, and why is the default target `"all"`? Seems like `""` might be a more reasonable default?

Reply:
- It's in-line because that's what some of the other events used. Though it appears our data classes are pretty inconsistent. I'm not opposed to switching the format.
- As for the `""`, I don't feel strongly either way. My only reasoning was that running a `/reload` by itself will trigger a reload of everything which is the equivalent of `/reload all`.